pages/index.js

```diff
@@ -37,22 +38,36 @@ function Event(props) {
 export default function Home() {
   return (
     <>
+      {/* Hero Banner */}
       <div className={`${styles.heroBanner}`}>
         <div className={`${styles.heroGrid}`}>
-          <div className={`${styles.heroGridItem}`}>
-            <h1 className={`${styles.heroTitle}`} style={{marginBottom:"33px"}}>Enhance <br></br>Empower<br></br>Enable</h1>
-            <Button type="button" className={`${styles.heroButton}`} href="/get-involved">Get Involved ></Button>
-          </div>
-          <div className={`${styles.heroGridItem}`}>
+        <div className={`${styles.heroGridItem}`}>
           <img 
-            src='/assets/img/ICSSCFrame.png'
+            src='https://cdn.discordapp.com/attachments/507063798104981508/1020549649523814400/Subtract.png'
```

Comment:
This image link should break in the future. It should be saved into this project.